code/modules/power/lighting.dm

```diff
@@ -9,7 +9,8 @@
 #define LIGHT_BROKEN 2
 #define LIGHT_BURNED 3
 
-
+GLOBAL_VAR_INIT(broken_sparks_min, 30)
+GLOBAL_VAR_INIT(broken_sparks_max, 100)
```

Comment:
Change it to 150-300 and make this a define

Reply:
Those are very high values. I did test the ones I'm currently using, and my low key laptop could hope with a whole bunch of broken lights fine.

Reply:
Use the seconds define too for duration values. `15 SECONDS` `30 SECONDS` etc